Dockerfiles/agent/windows/entrypoint/entrypoint.cpp

```diff
@@ -209,8 +230,16 @@ int _tmain(int argc, _TCHAR** argv)
 
     try
     {
-        ExecuteInitScripts();
-        const std::wstring command = argv[1];
+        auto runInitScripts = GetEnvVar(L"ENTRYPOINT_INITSCRIPTS");
+        if (runInitScripts.empty() || runInitScripts.compare(TRUE_STR) == 0)
+        {
+            ExecuteInitScripts();
+        }
+
+        // We checked earlier that argc >= 2 if command is empty
```

Comment:
If you move this logic up to where `command` is defined, you probably don't need to clarify this on a comment.

Reply:
Indeed, but I prefer to keep the logic close to where it's used and the early exit also makes sense so ended up splitting the two